components/PlantDetails/index.js

```diff
@@ -71,20 +64,38 @@ export default function PlantDetails({
           <br />
           <h4>Fertiliser Seasons:</h4>
           <ul>
-            {fertiliserSeason.map((season) => (
+            {plant.fertiliserSeason.map((season) => (
               <li key={season}>{season} </li>
             ))}
           </ul>
           <br />
         </StyledPlantNeedsContainer>
         <StyledImageContainer>
-          <StyledImage src={imageUrl} alt={name} fill />
+          <StyledImage src={plant.imageUrl} alt={plant.name} fill />
         </StyledImageContainer>
       </StyledPlantContainer>
       <h4>Description:</h4>
-      <p>{description}</p>
+      <p>{plant.description}</p>
       <br />
-      <PlantDeleteButton onDeletePlant={onDeletePlant} name={name} id={id}  />
+      <EditDeleteButton buttonText={"Edit"} handleButtonFunction={() => handleToggleModal("Edit")} />
+      <EditDeleteButton
+        buttonText={"Delete"}
+        handleButtonFunction={() => handleToggleModal("Delete")}
+      />
+
+      {showModal &&
+        <Modal modalContent={
+          isEdit ? <Form handleAddPlant={handleAddPlant}
+            plant={plant}
+            buttonText={"Edit"}
+            handleToggleModal={handleToggleModal}
+            handleEditPlant={handleEditPlant}
+
+          /> :
+            isDelete ? <PlantDeleteSection plant={plant} buttonText={"Delete"} onDeletePlant={onDeletePlant} id={plant.id} handleToggleModal={handleToggleModal} /> :
+              "dddd"
```

Comment:
Please explain 🤔

Reply:
Oopsies, we forgot to write the error msg